includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -661,4 +661,21 @@ public function save_payment_method_checkbox( $force_checked = false ) {
 		</fieldset>
 		<?php
 	}
+
+	/**
+	 * Gets the transaction URL.
+	 * Overrides WC_Payment_Gateway::get_transaction_url().
+	 *
+	 * @param  WC_Order $order Order object.
+	 * @return string
+	 */
+	public function get_transaction_url( $order ) {
+		if ( $this->testmode ) {
+			$this->view_transaction_url = 'https://dashboard.stripe.com/test/payments/%s';
```

Comment:
Since now we are repeating this string, maybe we should consider moving it to a new constant (not a blocker).

Reply:
Good call @wjrosa! I moved common logic to `WC_Stripe_Helper` in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3383/commits/73c440e2d0461e8a26f286cea2b035df813a7abd, and smoke-tested the changes. LMK what you think!

Reply:
Thanks for changing it. Looks good!